google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -303,15 +303,17 @@ private LogEntry entryFor(LogRecord record) {
       return null;
     }
     Level level = record.getLevel();
-    Map<String, String> labels = ImmutableMap.of(
-        "levelName", level.getName(),
-        "levelValue", String.valueOf(level.intValue()));
-    return LogEntry.newBuilder(Payload.StringPayload.of(payload))
-        .setLabels(labels)
-        .setSeverity(severityFor(level))
-        .build();
+    LogEntry.Builder builder = LogEntry.newBuilder(Payload.StringPayload.of(payload))
+        .addLabel("levelName", level.getName())
+        .addLabel("levelValue", String.valueOf(level.intValue()))
+        .setSeverity(severityFor(level));
+    return buildEntryFor(record, builder);
   }
 
+  protected LogEntry buildEntryFor(LogRecord record, LogEntry.Builder builder) {
```

Comment:
Correct buildEntryFor is sufficient for our needs.
I made `entryFor` protected as my first approach was going to be to replace that method.  I then left it protected as I thought others might like to extend it.   But we don't need it.   

Shall I remove it from this PR?